pom.xml

```diff
@@ -240,7 +240,7 @@
         <slf4j.wso2.version>1.5.10.wso2v1</slf4j.wso2.version>
 
         <!-- Automation Framework version-->
-        <test.framework.version>4.4.6</test.framework.version>
+        <test.framework.version>4.4.12</test.framework.version>
```

Comment:
Is there any specific reason related to $subject to bump this version?

Reply:
The previous version encountered a `NullPointerException` during compilation, necessitating a version upgrade to resolve the issue.
Since the `product-is` master branch is using version 4.4.12, the upgrade was aligned with that version.